cli/config.go

```diff
@@ -33,21 +32,21 @@ func read(file string) (Config, error) {
 	return c, nil
 }
 
-func ParseConfig() {
+func ParseConfig() error {
 	if ConfigPath == "" {
 		// No config file
-		return
+		return errors.New("empty config path")
 	}
```

Comment:
@SammyOina, CLI is not working getting `empty config path`

Reply:
ok, i'll return nill instead

Reply:
@SammyOina At Two places we need to return nil 
https://github.com/SammyOina/mainflux/blob/code-quality/cli/config.go#L44
https://github.com/SammyOina/mainflux/blob/code-quality/cli/config.go#L38